seb_openedx/settings/common.py

```diff
@@ -29,7 +29,17 @@ def plugin_settings(settings):
     See: https://github.com/edx/edx-platform/blob/master/openedx/core/djangoapps/plugins/README.rst
     """
     settings.SEB_COURSE_MODULE = 'seb_openedx.edxapp_wrapper.backends.get_course_module_h_v1'
+    settings.SEB_COURSEWARE_MODULE = 'seb_openedx.edxapp_wrapper.backends.get_courseware_module_h_v1'
+    settings.SEB_XMODULE_MODULESTORE_MODULE = 'seb_openedx.edxapp_wrapper.backends.get_courseware_module_h_v1'
     settings.SEB_CONFIGURATION_HELPERS = 'seb_openedx.edxapp_wrapper.backends.get_configuration_helpers_h_v1'
     settings.SEB_EDXMAKO_MODULE = 'seb_openedx.edxapp_wrapper.backends.edxmako_module_h_v1'
     settings.SEB_PERMISSION_COMPONENTS = ['AlwaysAllowStaff', 'CheckSEBKeysRequestHash']
     settings.SEB_KEY_SOURCES = ['from_global_settings', 'from_other_course_settings', 'from_site_configuration']
+
+    # When SEB determines the access is denied one may specify what to whitelist/blacklist more granularly
+    settings.SEB_WHITELIST_PATHS = []
+    settings.SEB_BLACKLIST_CHAPTERS = []
```

Comment:
At some point this will be defined in the UI for each course, both SEB_WHITELIST_PATHS and SEB_BLACKLIST_CHAPTERS, correct?

Reply:
Yep